templates/jsonfeed/shell.tpl.php

```diff
@@ -72,48 +72,24 @@
 
             $feedItem['content_html'] = $item->getBody();
 
-            $feedItem['_indieweb'] = array();
+            $feedItem['_meta'] = array();
             if (method_exists($item, 'getMetadataForFeed')) {
-                $feedItem['_indieweb'] = $item->getMetadataForFeed();
-            } else if ($item instanceof \IdnoPlugins\Like\Like) {
+                $feedItem['_meta'] = $item->getMetadataForFeed();
```

Comment:
Any reason why these couldn't be pulled from jsonSerialize()? ... I'd prefer to avoid adding an extra method on classes if possible, or if it isn't possible to avoid, then we should probably add a sensible default to the Entity superclass, so we don't have to test for the method...